presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/StaticHiveCluster.java

```diff
@@ -65,13 +66,27 @@ public StaticHiveCluster(List<URI> metastoreUris, String metastoreUsername, Hive
     public HiveMetastoreClient createMetastoreClient()
             throws TException
     {
-        List<HostAndPort> metastores = new ArrayList<>(addresses);
-        Collections.shuffle(metastores.subList(1, metastores.size()));
```

Comment:
I think you should still keep `shuffle`

Reply:
But I want to keep the failed the last one